openstack-mixin/dashboards.libsonnet

```diff
@@ -18,23 +18,26 @@ local logslib = import 'logs-lib/logs/main.libsonnet';
         + g.dashboard.withPanels(
           g.util.grid.wrapPanels(
             [
-              panels.placementStatus { gridPos+: { w: 4 } },
-              panels.keystoneStatus { gridPos+: { w: 4 } },
-              panels.novaStatus { gridPos+: { w: 4 } },
-              panels.neutronStatus { gridPos+: { w: 4 } },
-              panels.cinderStatus { gridPos+: { w: 4 } },
-              panels.glanceStatus { gridPos+: { w: 4 } },
+              panels.placementStatus { gridPos+: { w: 4, h: 2 } },
```

Comment:
This change makes the panels harder to read. A slight reduction could be fine given a solid reason, but this is too small imo.